marklogic-data-hub-central/ui/e2e/cypress/support/pages/graphExplore.tsx

```diff
@@ -97,6 +97,26 @@ class GraphExplore {
     return cy.findByLabelText("graphViewRightArrow");
   }
 
+  getUnmergeIcon() {
+    return cy.findByTestId("unmergeIcon");
+  }
+
+  getUnmergeOption() {
+    return cy.findByTestId("UnmergeOption");
+  }
+
+  getUnmergeSpinner() {
+    return cy.findByTestId("hc-button-component-spinner");
+  }
+
+  getModal() {
```

Comment:
Can we rename it to know which modal from Explore we are finding?
Example: getMergeModal or getDocCompareModal, etc

Reply:
Done! :)